velox/exec/TraceConfig.h

```diff
@@ -16,6 +16,7 @@
 
 #pragma once
 
+#include <stdint.h>
```

Comment:
Can you please move the system include in alphabetical order?
Is this needed when you built with Clang?